bugbot/rules/perfalert_resolved_regression.py

```diff
@@ -87,6 +104,63 @@ def should_needinfo(self, bug_comments, status_time):
 
         return True
 
+    def get_resolution_comments(self, comments, status_time):
+        resolution_comment = None
+        preceding_comment = None
+
+        for comment in comments:
+            if comment["creation_time"] > status_time:
+                break
+            if comment["creation_time"] == status_time:
+                resolution_comment = comment
+            if (
+                comment["author"] != BOT_MAIN_ACCOUNT
+                and comment["author"] != "intermittent-bug-filer@mozilla.bugs"
+            ):
```

Comment:
What do you think about excluding bots in general? 

We could use the. `is_bot_email()` utility function for that: https://github.com/mozilla/bugbot/blob/88d1a52f2d2902be34e651efcb15e3807938fe50/bugbot/utils.py#L610